src/controller/java/AndroidCurrentFabricRemover.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <controller/CurrentFabricRemover.h>
+#include <jni.h>
+
+namespace chip {
+namespace Controller {
+
+/**
+ * A helper class that can be used by consumers that don't care about the callback from the
+ * remove Fabric process and just want automatic cleanup of the CurrentFabricRemover when done
```

Comment:
Uh... but this is meant to be used by someone who _does_ care about the callback, and passes in a jobject to receive it, no?  Please fix this documentation to match what the class is really doing.

Reply:
I see. I will change some sentense.